src/Module/EddBkRestApiModule.php

```diff
@@ -139,10 +139,26 @@ public function setup()
                     return new SessionsController(
                         $c->get('eddbk_rest_api_sessions_iterator_factory'),
                         $c->get('sessions_select_rm'),
+                        $c->get('eddbk_rest_api_sessions_ordering'),
                         $c->get('sql_expression_builder')
                     );
                 },
 
+                /**
+                 * The order to use for sessions in the REST API.
+                 *
+                 * @since [*next-version*]
+                 */
+                'eddbk_rest_api_sessions_ordering' => function (ContainerInterface $c) {
+                    return [
+                        $c->get('sql_order_factory')->make([
```

Comment:
In the future, when order objects have their own factories, new constants should be used for keys.
https://github.com/Dhii/sql-interface/issues/9